### PR TITLE
Updating ingress-nginx to v0.26.1

### PR DESCRIPTION
**Pull request by knangia**
this updates ingress-nginx from v025.1 to v0.26.1

Signed-off-by: Khanak Nangia khanak.nangia@intel.com

### CODE REVIEW

File: clr-k8s-examples/5-ingres-lb/overlays/nginx-0.26.1/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+resources:
+  - ingress-nginx/deploy/static/mandatory.yaml
+  - ingress-nginx/deploy/static/provider/baremetal/service-nodeport.yaml
+
```
Comment (ahsan518) [inline clr-k8s-examples/5-ingres-lb/overlays/nginx-0.26.1/kustomization.yaml]: Can you please remove this extra space.

Comment (knangia) [reply clr-k8s-examples/5-ingres-lb/overlays/nginx-0.26.1/kustomization.yaml]: you mean the extra line 4

Comment (ahsan518) [reply clr-k8s-examples/5-ingres-lb/overlays/nginx-0.26.1/kustomization.yaml]: Yes

Comment (ahsan518) [reply clr-k8s-examples/5-ingres-lb/overlays/nginx-0.26.1/kustomization.yaml]: Else looks good.

Comment (knangia) [reply clr-k8s-examples/5-ingres-lb/overlays/nginx-0.26.1/kustomization.yaml]: done, thanks!

**Pull request merged**